CODE_OF_CONDUCT.md

```diff
@@ -1,40 +1,36 @@
 
-This policy is NOT intended to supersede the Fuel Rats [Terms of Service](https://fuelrats.com/terms-of-service), but as a supplement to cover non-rescue contributions.
-
 ## Standards
 
 We hope all members strive to go beyond examples laid out here and actively create a positive and respectful atmosphere.
 
 * **Foster a friendly, judgement-free environment for all.**  This project houses international contributors from all walks of life, with varying experience.
-* **Keep an open-mind, and respect differing viewpoints.** Lest we forget the professional attitude best displayed in our work.
+* **Respect differing viewpoints.** Lest we forget the professional attitude best displayed in our work.
 * **Be professional and considerate.** Discuss and resolve conflicts in an open, civil manner.  Disagreements are inevitable.
 * **Show patience and be helpful.**  We all make mistakes.  Playing the 'blame game' helps no one.
 
 #### Negative Behavior Examples
 
+These examples should not be considered an exhaustive list.  This code of conduct establishes the parameters of expected conduct, and contributors or contributions that fall outside that spirit are not welcome.
+
+* Discriminatory behavior
 * Forcing others to participate
 * Public or private harassment
 * Trolling/Personal attacks
 * 'Doxxing' or making a person's private information available without explicit permission
 * Conduct considered inappropriate for a professional environment.
-* Conduct considered unbecoming by any member of the 'Tech Rat Moderation' team.
 
-## Repository Admins/Owners
+## Project Leaders
 
-Repository Owners are expected to use this document or provide a link to it in their repository. Additional items may be added, but no thing may be added to reduce the effectiveness of this document or its intent.
-
-Admins/Owners have the responsibility to remove and report comments, code, commits, issues, or any other post that does not align with this policy, and report it to a member of the 'Tech Rat Moderation' team.
+Tech Rats who maintain a repository have the responsibility to remove and report comments, code, commits, issues, or any other post that does not align with this policy, and report it to a member of the 'Tech Rat Moderation' team.
 
 ## Scope
-The scope of this document applies to everyone within the organization's project space, and any individual representing the organization in public spaces.
+The scope of this document applies to everyone within the Fuel Rats project spaces, any use of an official Fuel Rats account, or when individually speaking as a representative of the Fuel Rats.
```

Comment:
Again I bring up the point that we could have people complain because they interpret this as

> well they're an org member, therefore they're always a representative of the Fuel Rats.

Yes us moderators can quickly shoot this down, but it's still causing a confusion

Reply:
I'm open to any and all suggestions.  That was just the most recent suggested wording.

Reply:
use the wording `official representative`? just becuase they are a member of TFRM doesn't make them a representative.

Reply:
'officially representing' or 'specifically representing... as a whole' works for me, or other grammatical equivalents.